client/src/store/TODOList/thunk.js

```diff
@@ -61,3 +61,5 @@ export default {
   editTODOItemAsync,
   deleteTODOItemAsync,
```

Comment:
I vaguely recall that we decided to replace the delete button that shows up on each TODO List card with a button to mark the TODO as finished since people are more likely to mark a TODO item as complete rather than delete it outright (in case it turns out they didn't finish it, to encourage keeping a history of TODOs so people can look back at what they did, etc.) Currently the user must open a modal every time they want to mark a TODO as finished.

Reply:
yeah.... I did not have time to get to that. This will have to be done after the milestone 5 demo

Reply:
But this is a refactor that will be made :)

Reply:
Main thing is just to have the ability to mark a TODO as finished upfront: the isFinished checkbox can probably still be in the update modal. The delete button could even remain on the list card view as long as the ability to mark it as finished is also there or exists elsewhere.

Reply:
If you don't want to have the isFinished checkbox in the update modal can make it so the PUT request never changes it (request always sent out with same isFinished as before) or change the request to a PATCH request with input validation instead.

Reply:
Given the amount of change this might entail, I propose that this change be added to tasks to do after the milestone 5 demo. Would that work?